p2p/message_test.go

```diff
@@ -22,10 +22,10 @@ import (
 	"testing"
 
 	"github.com/ChainSafe/gossamer/common"
+	"github.com/ChainSafe/gossamer/common/optional"
 )
 
-func TestDecodeMessage_Status(t *testing.T) {
-	// from Alexander testnet
+func TestDecodeMessageStatus(t *testing.T) {
```

Comment:
There are still a number of "magical" values in here. i know they're a pain to document but I promise you will be thankful if you ever need to change one :pray:

Reply:
the status messages came directly from the testnet, I can still document them if you'd like

Reply:
Yeup, that would helpful too

Reply:
I added a comment but idk if it's helpful, since it basically repeats the exact thing that's a few lines down that compares the values